tests/CMakeLists.txt

```diff
@@ -199,6 +199,13 @@ target_link_libraries(su3_test ${TEST_LIBS})
 quda_checkbuildtest(su3_test QUDA_BUILD_ALL_TESTS)
 install(TARGETS su3_test ${QUDA_EXCLUDE_FROM_INSTALL} DESTINATION ${CMAKE_INSTALL_BINDIR})
 
+if(QUDA_DIRAC_LAPLACE)
```

Comment:
I believe this also needs to be guarded around making sure some Wilson-type (nSpin = 4) dslash:
```
if(QUDA_DIRAC_WILSON
   OR QUDA_DIRAC_CLOVER
   OR QUDA_DIRAC_TWISTED_MASS
   OR QUDA_DIRAC_TWISTED_CLOVER
   OR QUDA_DIRAC_NDEG_TWISTED_MASS
   OR QUDA_DIRAC_NDEG_TWISTED_CLOVER
   OR QUDA_DIRAC_DOMAIN_WALL)
   ```

Because this test assumes an nSpin=4 field. I can pop that in, just noting it here for now.

Reply:
I have guarded the ctest with the WILSON condition, and also added a check in the test for enablement of nSpin 4.